cpmpy/expressions/utils.py

```diff
@@ -109,3 +109,12 @@ def eval_comparison(str_op, lhs, rhs):
         raise Exception("Not a known comparison:", str_op)
 
 
+def get_bounds(expr):
+    # can return floats, use floor and ceil when creating an intvar!
+    if hasattr(expr,'get_bounds'):
```

Comment:
I am not convinced by this.

I think it would be clearer to do
```if isinstance(expr): return expr.get_bounds(); else return expr,expr```?

Reply:
This would lead to circular import for Expression, I put the import in the get_bounds function